### PR TITLE
[Maybe revisit] Feature/pno 12 - draft implementation of elements, pseudo_sets 

### PR DESCRIPTION
**Pull request by azech-hqs**

### CODE REVIEW

File: example/methods_directory_poc/physical/qm/pseudopotential/psp_file.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            "value": 1.5,
+            "units": "bohr"
+        },
+        "cutoff": {
+            "value": 1.9,
+            "units": "bohr"
+        }
+    },
+    "zValence": 19,
+    "modelReference": "pb/qm/dft/ks-dft/gga#hse06?alphac=0.1",
```
Comment (dmytro-spodarets) [inline example/methods_directory_poc/physical/qm/pseudopotential/psp_file.json:73]: Might be better to use object?

Comment (azech-hqs) [reply example/methods_directory_poc/physical/qm/pseudopotential/psp_file.json:73]: The model "URL" serves three purposes at once (is human-readable, shows provenance, is unique). While this example is very compact, the notation can become very long for strongly modified models. Also, this model URL only points to a unit model, but we may want to point to a composite model instead. As an alternative we could use an object where the purposes are separated. Perhaps something like the following:
```json
"modelReference": {
    "id": "abcdefg123456",
    "types": ["pb/qm/dft/ks-dft/gga/hse06"],
    "isModified": true
}
```

File: schema/definitions/science.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "relativity": {
+        "enum": [
+            "none",
+            "1c-scalar",
+            "2c-quasi",
+            "4c-full"
+        ]
+    },
+    "elements": {
```
Comment (timurbazhirov) [inline schema/definitions/science.json:10]: 1. How do we deal with periodic-table.js now, with the list of elements defined here?
2. The current placing in `science.json` seems a bit ad-hoc, maybe we better off isolating this work item (regarding where to put elements) to a future task
3. As we discussed before, "elements" is an overloaded word in web development, I'd suggest spelling "chemicalElementSymbols" is the future

File: schema/methods_directory_poc/physical/qm/pseudopotential/psp_set.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        "source": {
+            "description": "explains the source of where this came from",
+            "$ref": "psp_file/origin.json#/definitions/ppSources"
+        },
+
+        "version": {
+            "description": "explains the version of where this came from",
+            "type": "string"
+        },
+        "files": {
```
Comment (timurbazhirov) [inline schema/methods_directory_poc/physical/qm/pseudopotential/psp_set.json:43]: This is against the logic we used in the entity_sets to entity relationship - see https://github.com/Exabyte-io/code.js/blob/main/src/entity/set/mixins.js#L9, for instance. Currently the entities (i.e. psp_files in the current context) are pointing to the set (psp_set) by having a reference to a set `_id` in their `inSet` field.

This is a good effort, but I think we should come back to defining the way sets are handled at a later point.

File: schema/methods_directory_poc/physical/qm/pseudopotential/psp_file.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Schema for pseudopotential files",
```
Comment (timurbazhirov) [inline schema/methods_directory_poc/physical/qm/pseudopotential/psp_file.json:3]: Do we intentionally omit schema id in this PR?

**Comment by timurbazhirov** [comment]
Adjusted the name, closing per our discussion

**Pull request closed without merging**